README.md

```diff
@@ -172,284 +159,36 @@ The TraverseEngine provides some methods for graph traversal:
-        "type": "Inheritance",
-        "template": ["Inheritance", "Concept", "Concept"],
-        "targets": [
-            {
-                "handle": "af12f10f9ae2002a1607ba0b47ba8407",
-                "type": "Concept",
-                "name": "human",
-            },
-            {
-                "handle": "bdfe4e7a431f73386f37c6448afe5840",
-                "type": "Concept",
-                "name": "mammal",
-            },
-        ],
-    }
-]
-```
-
-#### Remote scope synchronized with local Atoms
+## Tests
 
-```python
+In the main project directory, you can run the command below to run the unit and integration tests
 
-# Add a local Link
-das.add_link({
-    'type': 'Inheritance',
-    'targets': [
-        {'type': 'Concept', 'name': 'monkey'},
-        {'type': 'Concept', 'name': 'mammal'}
-    ]
-})
-
-query = {
-    'atom_type': 'link',
-    'type': 'Inheritance',
-    'targets': [
-        {'atom_type': 'node', 'type': 'Concept', 'name': 'human'},
-        {'atom_type': 'node', 'type': 'Concept', 'name': 'mammal'},
-    ]
-}
-
-query_params = {
-    "toplevel_only": False,
-    "query_scope": "synchronous_update"
-}
-
-resp = das.query(query, query_params)
-
-print(resp)
-```
 ```bash
-[
-    {
-        "handle": "c93e1e758c53912638438e2a7d7f7b7f",
-        "type": "Inheritance",
-        "template": ["Inheritance", "Concept", "Concept"],
-        "targets": [
-            {
-                "handle": "af12f10f9ae2002a1607ba0b47ba8407",
-                "type": "Concept",
-                "name": "human",
-            },
-            {
-                "handle": "bdfe4e7a431f73386f37c6448afe5840",
-                "type": "Concept",
-                "name": "mammal",
-            },
-        ],
-    },
-    {
-        "handle": "f31dfe97db782e8cec26de18dddf8965",
-        "type": "Inheritance",
-        "template": ["Inheritance", "Concept", "Concept"],
-        "targets": [
-            {
-                "handle": "1cdffc6b0b89ff41d68bec237481d1e1",
-                "type": "Concept",
-                "name": "monkey",
-            },
-            {
-                "handle": "bdfe4e7a431f73386f37c6448afe5840",
-                "type": "Concept",
-                "name": "mammal",
-            },
-        ],
-    },
-]
+make unit-tests
 ```
 
-### Traverse
-
-```python
-from hyperon_das import DistributedAtomSpace
-
-das = DistributedAtomSpace(query_engine='remote', host='192.32.11.45', port=9000)
-
-traverse = das.get_traversal_cursor(handle='12345', handles_only=True)
-
-
-current_cursor = traverse.get()
-
-links = traverse.get_links(link_type='Similarity', cursor_position=0, target_type='Concept')
-
-neighbors = traverse.get_neighbors(link_type='Inheritance', target_type='Concept')
-
-traverse.follow_link(link_type='Similarity', unique_path=True)
-
-traverse.goto(handle='9990')
-
-```
-
-## Tests
-
-You can run the command below to run the unit tests
-
 ```bash
-make test-unit
+make integration-tests
```

Comment:
Add an explanation of how integration tests use a remote server to execute. Provide server IP etc. List knowledge base contents (in MeTTa)

Reply:
Done!